qiskit/circuit/parameterexpression.py

```diff
@@ -18,14 +18,16 @@
 from dataclasses import dataclass
 from enum import IntEnum
 from typing import Callable, Union
-
 import numbers
 import operator
 
 import numpy
-import symengine
 
 from qiskit.circuit.exceptions import CircuitError
+import qiskit._accelerate.circuit
+
+SymbolExpr = qiskit._accelerate.circuit.PySymbolExpr
```

Comment:
note that not all pyfunctions appear in this file (e.g. pow). 
Convenience functions for add, sub, mul, div, pow, rsub, rdiv, rpow are not documented. See https://github.com/Qiskit/qiskit/issues/13510

Reply:
`SymbolExpr` isn't a public class, it's only internally used for `ParameterExpression`. The dunder methods from #13510 we can adding docs for those independently. Exactly how we document those is actually unclear to me as `__pow__` isn't something users run themselves they do `expr ** 5` which is what `__pow__` provides. So I'm not sure adding docstrings for the dunder methods is all that useful for people.

Reply:
the problem is that it's not clear to the users that they can use `expr ** 5` or `expr1 * expr2` etc.